pkg/network/tracer/tracer.go

```diff
@@ -119,7 +117,7 @@ func NewTracer(config *config.Config) (*Tracer, error) {
 
 // newTracer is an internal function used by tests primarily
 // (and NewTracer above)
-func newTracer(cfg *config.Config) (*Tracer, error) {
+func newTracer(cfg *config.Config) (rettr *Tracer, reterr error) {
```

Comment:
I don't see `rettr` used anywhere, and `reterr` seems to not be set anywhere.

Reply:
`reterr` is automatically set if you use `return`

Reply:
Also, if you name one return value, you have to name them all.

Reply:
I think `(_ *Tracer, reterr error)` works.